src/asr/asr_mc_chainer.py

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+
+import collections
+import copy
+import json
+import logging
+import math
+import os
+import pickle
+import six
+import sys
+
+# chainer related
+import chainer
+from chainer import cuda
+from chainer import function
+from chainer import reporter as reporter_module
+from chainer import training
+from chainer.training import extensions
+
+# espnet related
+from asr_utils import adadelta_eps_decay
+from asr_utils import CompareValueTrigger
+from asr_utils import delete_feat
+from asr_utils import make_batchset
+from asr_utils import restore_snapshot
+from e2e_asr_attctc import E2E
+from e2e_asr_attctc import Loss
+
+# for me2e
+from beamformer import NB_MVDR
+from me2e_asr_attctc import ME2E
+
+# for kaldi io
+import kaldi_io_py
+import lazy_io
+
+# rnnlm
+import lm_train
+
+# numpy related
+import matplotlib
+import numpy as np
+matplotlib.use('Agg')
+
+
+class ChainerSeqEvaluaterKaldi(extensions.Evaluator):
```

Comment:
Is this class completely same to the one in `asr_chainer.py`? If it is true, import and use it.